internal/network/command_create.go

```diff
@@ -14,29 +14,33 @@ import (
 
 func (c *command) newCreateCommand() *cobra.Command {
 	cmd := &cobra.Command{
-		Use:   "create <name>",
+		Use:   "create [name]",
 		Short: "Create a network.",
-		Args:  cobra.ExactArgs(1),
+		Args:  cobra.MaximumNArgs(1),
 		RunE:  c.create,
 		Example: examples.BuildExampleString(
 			examples.Example{
 				Text: `Create a Confluent network in AWS with connection type "transitgateway" by specifying zones and CIDR.`,
+				Code: "confluent network create --cloud aws --region us-west-2 --connection-types transitgateway --zones usw2-az1,usw2-az2,usw2-az4 --cidr 10.1.0.0/16",
+			},
+			examples.Example{
+				Text: `Create a Confluent network with a display name in AWS with connection type "transitgateway" by specifying zones and CIDR.`,
```

Comment:
```suggestion
				Text: `Create a named Confluent network in AWS with connection type "transitgateway" by specifying zones and CIDR.`,
```

Reply:
Updated and for other sub-commands as well.